### PR TITLE
Added support for ktables

### PR DESCRIPTION
**Pull request by cmebarrow**
Requires https://github.com/reaktivity/nukleus-kafka.spec/pull/12

### CODE REVIEW

File: src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java [hunk 1]
```diff
@@ -345,43 +346,49 @@ private void doKafkaData(
...
+                                final long offset = fetchOffsets.get(partition);
+                                a.item(p -> p.set(offset));
+                            }
+                        }
+                    });
+            if (messageKey != null)
+            {
+                dataExRW.messageKey(messageKey.buffer(), messageKey.offset(), messageKey.sizeof());
+            }
+            return dataExRW.build()
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java]: if `dataExRW.messageKey(null)` works using `KafkaDataEx.Builder.messageKey(OctetsFW messageKey)` method, then let's use `dataExRW.messageKey(messageKey)` unconditionally here and keep the builder flow intact.

File: src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java [hunk 2]
```diff
@@ -70,7 +70,8 @@
 
     private final KafkaRouteExFW routeExRO = new KafkaRouteExFW();
     private final KafkaBeginExFW beginExRO = new KafkaBeginExFW();
-    private final OctetsFW octetsRO = new OctetsFW();
+    private final OctetsFW octetsRO1 = new OctetsFW();
+    private final OctetsFW octetsRO2 = new OctetsFW();
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java]: Suggest renaming to `valueRO` and `keyRO`.

File: src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java (hunk 1 above)
Comment (cmebarrow) [reply src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java]: thanks, done

File: src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java (hunk 2 above)
Comment (cmebarrow) [reply src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java]: done

**Pull request merged**